src/ch/blinkenlights/android/vanilla/PlaybackService.java

```diff
@@ -385,6 +392,10 @@
 	private int mReplayGainBump;
 	private int mReplayGainUntaggedDeBump;
 	/**
+	 * Percentage to set the volume as while a notification is playing (aka ducking)
+	 */
+	private int mVolumeOnNotification;
+	/**
```

Comment:
Yip, no problem. I will leave the preference text itself as 'Volume On Notification', so users don't have to know what ducking is?